modules/nw-networkpolicy-create-cli.adoc

```diff
@@ -121,6 +124,95 @@ where
 `<network_name>`:: Specifies the name of a network attachment definition.
 --
 endif::multi[]
++
+.Allow ingress traffic to one pod from a particular namespace
++
+This policy allows traffic to pods labelled `pod-a` from pods running in `namespace-y`.
++
+[source,yaml]
+----
+ifndef::multi[]
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+endif::multi[]
+ifdef::multi[]
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+endif::multi[]
+metadata:
+  name: allow-traffic-pod
+ifdef::multi[]
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: <network_name>
+endif::multi[]
+spec:
+  podSelector:
+   matchLabels:
+      pod: pod-a
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+           kubernetes.io/metadata.name: namespace-y
+----
+ifdef::multi[]
++
+--
+where
+
+`<network_name>`:: Specifies the name of a network attachment definition.
```

Comment:
I'm not sure, but I don't think this is rendering correctly. Also specified is capitalized and I don't think it should be.

Reply:
"Specifies" should be capitalized.